pkgs/native_synchronization/pubspec.yaml

```diff
@@ -0,0 +1,14 @@
+name: native_synchronization
+description: Low level synchronization primitives built on dart:ffi.
+version: 0.3.0
+repository: https://github.com/dart-lang/native_synchronization
```

Comment:
Update to `labs/`

Reply:
sgtm; will plan to do this as part of a follow-up PR